uuid.gd

```diff
@@ -1,32 +1,29 @@
+# Note: The code might not be as pretty it could be, since it's written
+# in a way that maximizes performance. Methods are inlined and loops are avoided.
 extends Node
 
-static func getRandomInt(max_value):
-  randomize()
-
-  return randi() % max_value
-
-static func randomBytes(n):
-  var r = []
-
-  for index in range(0, n):
-    r.append(getRandomInt(256))
-
-  return r
-
 static func uuidbin():
```

Comment:
I left `uuidbin()` here as I suppose some users may want to use it. If that's not the case, the method can be removed entirely since it's not called anywhere.

Reply:
> If I dont call `randomize` there, then there will be collisions (you might need to generate 1 000 000 ids to get one)

That sounds like a bug. Please report it on the Godot issue tracker :slightly_smiling_face:

Either way, are there cases where the user will generate a large amount of UUIDs without being able to call `randomize()` on a regular basis? I call `randomize()` every time an UUID is generated, but this makes the new method 30% slower.